src/main/java/duke/command/DukeActions.java

```diff
@@ -1,10 +1,6 @@
 package duke.command;
 
-import duke.util.DukeDB;
-import duke.util.DukeException;
-import duke.util.DukeConfig;
-import duke.util.DukeTaskList;
-import duke.util.Ui;
+import duke.util.*;
```

Comment:
maybe revert this change? discouraged to use * import

Reply:
fixed in ca26cb67916444729632edba7164a8834ff5f6e2